tests/sentry/integrations/github/test_integration.py

```diff
@@ -9,21 +8,35 @@
 import sentry
 from sentry.constants import ObjectStatus
 from sentry.integrations.github import API_ERRORS, GitHubIntegrationProvider
-from sentry.integrations.utils.code_mapping import CodeMapping, CodeMappingTreesHelper, Repo
 from sentry.models import Integration, OrganizationIntegration, Project, Repository
 from sentry.plugins.base import plugins
 from sentry.plugins.bases import IssueTrackingPlugin2
 from sentry.shared_integrations.exceptions import ApiError
 from sentry.testutils import IntegrationTestCase
-from sentry.utils import json
 
 TREE_RESPONSES = {
     "foo": {
         "status_code": 200,
         "body": {
             # The latest sha for a specific branch
             "sha": "a4e587563cb5dbb46192b5962cbadc8c532a8455",
-            "tree": [],
+            "tree": [
```

Comment:
I have changed the last test to focus on just generating a tree for every repo of an org, thus, reverting the fixture to a simpler version of it.